src/resolver.js

```diff
@@ -36,11 +40,11 @@ export default function resolve({http, fetch, spec, url, baseDoc, mode, allowMet
   }
   else {
     // Store the spec into the url provided, to cache it
-    plugins.refs.docCache[baseDoc] = spec
+    docCache[baseDoc] = spec
```

Comment:
cacheing is hard.
I don't mind putting it inside specmap... as to the cache. I would prefer we avoided a global. At the same time, I don't know of the performance of recreating the cache each time.
Your call on that.